src/polymer-cli.ts

```diff
@@ -12,6 +12,9 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
+// Bail out if invalid node version
+require('please-upgrade-node')(require('../package.json'));
```

Comment:
Doh! I definitely didn't look here.

I put this in the wrong place apparently. The require before import is to ensure that older versions that don't support `import`. By placing it in bin/polymer.js it is technically before all the other imports. 

I will modify this to do the bin/polymer.js